From 88033ad64b3597f3cdc1b3c101b42076687e0c6a Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Tue, 18 Feb 2014 16:03:18 -0500
Subject: [PATCH 14/14] Add missing +x

Without +x, this script will never get run by dib-run-parts.

Change-Id: I12c3639237b32943a02ff8ae6ab939784388d89a
---
 .../install.d/neutron-package-install/80-neutron-dhcp-agent               | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 elements/neutron-dhcp-agent/install.d/neutron-package-install/80-neutron-dhcp-agent

diff --git a/elements/neutron-dhcp-agent/install.d/neutron-package-install/80-neutron-dhcp-agent b/elements/neutron-dhcp-agent/install.d/neutron-package-install/80-neutron-dhcp-agent
old mode 100644
new mode 100755
-- 
1.8.5.3

